src/Markdown.js

```diff
@@ -56,23 +57,47 @@ export default class Markdown {
         return is_plain;
     }
 
-    toHTML() {
+    render(html) {
```

Comment:
Also, it is not at all clear that `html` is a `boolean`. Please document what it is supposed to do.